wemulate/controllers/show_controller.py

```diff
@@ -75,61 +78,63 @@ def _construct_interface_data_to_render(
-    typer.echo(
-        tabulate.tabulate(render_data, headers=CONNECTION_HEADERS, tablefmt="grid")
-    )
+    table = Table(title="Connection Information")
+    for header in CONNECTION_HEADERS:
+        table.add_column(header)
+    _populate_connection_table(connection, table)
+    console.print(table)
+    raise typer.Exit()
 
 
 @app.command(help="show overview about all connections")
 def connections():
     connections: List[ConnectionModel] = utils.get_connection_list()
     if not connections:
-        typer.echo("There are no connections")
-        raise typer.Exit()
+        err_console.print("There are no connections")
+        raise typer.Exit(1)
     else:
-        render_data: List = []
+        table = Table(title="Connection Information")
+        for header in CONNECTION_HEADERS:
+            table.add_column(header)
         for connection in connections:
-            _construct_connection_data_to_render(connection, render_data)
-        typer.echo(
-            tabulate.tabulate(render_data, headers=CONNECTION_HEADERS, tablefmt="grid")
-        )
+            _populate_connection_table(connection, table)
+        console.print(table)
+    raise typer.Exit()
 
 
 @app.command(help="show specific interface information", no_args_is_help=True)
 def interface(interface_name: str = typer.Argument(..., help="name of the interface")):
     if not interface_name in settings.get_non_mgmt_interfaces():
-        typer.echo("The given interface is not available")
+        err_console.print("The given interface is not available")
+        raise typer.Exit(1)
     else:
-        render_data: List = []
-        _construct_interface_data_to_render(
-            render_data,
-            interface_name,
-        )
-        typer.echo(
-            tabulate.tabulate(render_data, headers=INTERFACE_HEADER, tablefmt="grid")
-        )
+        table = Table(title=f"Interface [b]{interface_name}[/]")
+        for header in INTERFACE_HEADER:
+            table.add_column(header)
+        _populate_interface_table(table, interface_name)
+        console.print(table)
+    raise typer.Exit()
 
 
 @app.command(help="show overview about all interfaces")
 def interfaces():
-    render_data: List = []
+    table = Table(title="Interfaces")
+    for header in INTERFACE_HEADER:
+        table.add_column(header)
+
     for interface in settings.get_non_mgmt_interfaces():
-        _construct_interface_data_to_render(render_data, interface)
-    typer.echo(
-        tabulate.tabulate(render_data, headers=INTERFACE_HEADER, tablefmt="grid")
-    )
+        _populate_interface_table(table, interface)
+    console.print(table)
+    raise typer.Exit()
 
 
 @app.command(help="show overview about all management interfaces")
 def mgmt_interfaces():
     mgmt_interfaces: List[str] = settings.get_mgmt_interfaces()
-    render_data: List = []
+    table = Table(title="Management Interfaces")
+    for header in ["NAME", "IP", "MAC"]:
```

Comment:
Maybe we can add a constant for the management interface header for consistency reasons.